feature-group-definitions/aborting.dist.yml

```diff
@@ -0,0 +1,28 @@
+# Generated from: aborting.yml
+# Do not edit this file by hand. Edit the source file instead!
+# Last modified: 2024-04-04T09:35:23.902323657+00:00[UTC]
+
+name: AbortController and AbortSignal
+description: The `AbortController` and `AbortSignal` interfaces allow you to cancel an ongoing operation, such as a `fetch()` request.
+spec: https://dom.spec.whatwg.org/#aborting-ongoing-activities
+caniuse: abortcontroller
+status:
+  baseline: high
+  baseline_low_date: 2019-03-25
+  baseline_high_date: 2021-09-25
+  support:
+    chrome: "66"
+    chrome_android: "66"
+    edge: "16"
+    firefox: "57"
```

Comment:
Shouldn't this be 53? https://developer.mozilla.org/en-US/docs/Web/API/AbortController#browser_compatibility

Reply:
Opera is 53. Already, the machines are reading the tables better than we are 😄

Reply:
Excellent!